RNCalendarEvents.m

```diff
@@ -680,7 +675,11 @@ - (NSDictionary *)serializeCalendarEvent:(EKEvent *)event
         [formedCalendarEvent setValue: structuredLocation forKey:@"structuredLocation"];
     }
 
-    return [formedCalendarEvent copy];
+        return [formedCalendarEvent copy];
+    } @catch (NSException *exception) {
+        return nil;
+    }
```

Comment:
I am not familiar is iOS code, but this look like it's not that safe to return nil.
What is the goal of this? How does JS should behave with this change?